doc/rel_notes/1.0.0.txt

```diff
@@ -29,7 +29,10 @@ Build System
 
  * Simplified string logic for finding compiler flags file.
 
- * Added Eigen3 include directory path to include directories exported by MAP-Tk
+ * Added Eigen3 include directory path to include directories exported by
+   MAP-Tk.
+
+ * Added support for LIB_SUFFIX.
```

Comment:
... as opposed to "MAPTK_LIB_SUFFIX"
Just to note what its replacing in case someone was already using that.